README.rst

```diff
@@ -52,6 +51,23 @@ environ_config: Configuration with env variables for Python.
   >>> cfg.db.password
   's3kr3t'
 
+`AppConfig` instance can be also created by means of `environ.to_config` method:
```

Comment:
This is RST, so you'll have to use double backticks for code. :)

You don't have to add a whole example, it becomes hard to read. Just something like:

> `AppConfig.from_environ({...})` is equivalent to the code above, depending on your taste.

Reply:
Great use of [Occam's razor](https://en.wikipedia.org/wiki/Occam%27s_razor). Done.